Stanford/JumpstartEngineering/Install/CAPx/CAPxDisplay.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * @file
+ * Define the context for the CAP display view mode for person pages.
+ */
+
+namespace Stanford\JumpstartEngineering\Install\CAPx;
+/**
+ *
+ */
+class CAPxDisplay extends \AbstractInstallTask {
```

Comment:
I am at a toss up for this one. 

I don't want all of the dependencies to be spread out and scattered throughout the install tasks. I would rather have them all listed in the .info file of the profile but if we are to share these tasks with other products/people, it would be nice if we declared the dependencies. Maybe just a code comment?